device/mellanox/x86_64-mlnx_msn4700-r0/ACS-MSN4700/buffers_defaults_t1.j2

```diff
@@ -0,0 +1,91 @@
+{% set default_cable = '5m' %}
+{% set ingress_lossless_pool_size =  '19283968' %}
+{% set ingress_lossy_pool_size =  '19283968' %}
+{% set egress_lossless_pool_size =  '67108864' %}
```

Comment:
@keboliu we need to recalculate buffers size. You have used incorrect value (64 mb)

Reply:
fixed